turbo.json

```diff
@@ -21,9 +21,6 @@
 			"dependsOn": ["^build"],
 			"outputs": []
 		},
-		"prisma-generate": {
```

Comment:
My intention here was to remote prisma-generate from turbo because we don't ever want turbo to run it as far as I can tell. It also had `cache: false` anyway.